### PR TITLE
add a quick note about the wrong name change

### PR DESCRIPTION
**Pull request by pvinis**
Until we can just regenerate the diffs

# Summary

## Checklist

- [ ] I tested this thoroughly
- [ ] I added the documentation in `README.md` (if needed)

### CODE REVIEW

**Comment by lucasbento** [comment]
Isn't it better to manually delete the change on `rn-diff-purge`?

If not, I would rather add something to not render the diff for those files.

**Comment by pvinis** [comment]
Ugh, it is, but at least for now we can have it there too, so people know what's up? I'll do a quick script to remove it from the diffs

File: src/releases/0.60.js [hunk 1]
```diff
@@ -37,6 +37,26 @@ export default {
           for an explanation and some help with upgrading this file.
         </Fragment>
       )
-    }
+    },
+    {
+      fileName: 'ios/RnDiffApp/Info.plist ',
```
Comment (lucasbento) [inline src/releases/0.60.js]: Can you remove the ending space on the file names?

Comment (pvinis) [reply src/releases/0.60.js]: oopsssss

**Pull request merged**